certified-connectors/Oneflow/apiDefinition.swagger.json

```diff
@@ -265,11 +265,11 @@
         }
       }
     },
-    "/contracts/{contract_id}/": {
-      "get": {
-        "summary": "Get contract details",
-        "description": "Retrieve the details of a contract by its ID.",
-        "operationId": "GetContract",
+    "/contracts/{contract_id}/data_fields/c::{data_field_id}": {
+      "put": {
+        "summary": "Update a datafield",
```

Comment:
@DmitryKleshchov in some properties you use underscore for `data_fields` to emphasize that these are two separate words, also you use `data fields` string in script.csx file, while in apiDefinition you use `datafield`.

Please make sure everything is grammatically correct and is free of typographical errors.

You can refer to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

Reply:
@v-EgorKozhadei done